framework/src/org/checkerframework/common/value/ValueAnnotatedTypeFactory.java

```diff
@@ -947,18 +965,47 @@ private AnnotationMirror createNumberAnnotationMirror(List<Number> values) {
                 "ValueAnnotatedTypeFactory: unexpected class: " + first.getClass());
     }
 
+    /**
+     * Returns the set of possible values. Returns the empty list if no values are possible (for
+     * dead code). Returns null if any value is possible -- that is, if no estimate can be made --
+     * and this includes when there is no constant-value annotation so the argument is null.
+     */
     public static List<Long> getIntValues(AnnotationMirror intAnno) {
```

Comment:
Why is the method called `getIntValues` if it returns a list of Long? Can you add a sentence to the javadoc to motivate this?
It would also be nice to reference the set of AnnotationMirrors that are supported.